homeassistant/components/ambient_station/__init__.py

```diff
@@ -286,6 +286,35 @@
     return True
 
 
+async def async_migrate_entry(hass, config_entry):
+    """Migrate old entry."""
+    version = config_entry.version
+
+    _LOGGER.debug('Migrating from version %s', version)
+
+    reasons_to_re_add = {
+        1: "The format for unique IDs (in the entity registry) has changed."
+    }
+
+    # 1 -> 2: Unique ID format changed:
+    if version == 1:
+        config_entry.version = 2
+        hass.config_entries.async_update_entry(
+            config_entry, data=config_entry.data)
```

Comment:
You don't need to specify `data=config_entry.data` if you intend to save the data of the current entry itself.  This parameter is only needed if you intend to _replace_ the data dictionary entirely.

```suggestion
            config_entry)
```